maps-compose/src/main/java/com/google/maps/android/compose/RememberComposeBitmapDescriptor.kt

```diff
@@ -47,8 +51,8 @@ private fun renderComposableToBitmapDescriptor(
     composeView.draw(fakeCanvas)
 
     composeView.measure(
-        View.MeasureSpec.makeMeasureSpec(parent.width, View.MeasureSpec.AT_MOST),
-        View.MeasureSpec.makeMeasureSpec(parent.height, View.MeasureSpec.AT_MOST),
+        View.MeasureSpec.UNSPECIFIED,
+        View.MeasureSpec.UNSPECIFIED,
```

Comment:
I don't think this is correct, I believe it would need to be
```suggestion
        View.MeasureSpec.makeMeasureSpec(0, View.MeasureSpec.UNSPECIFIED),
        View.MeasureSpec.makeMeasureSpec(0, View.MeasureSpec.UNSPECIFIED),
```

Should be ok to makeMeasureSpec() just once in a top-level field.